lib/Controller/PreviewController.php

```diff
@@ -129,7 +129,12 @@ public function index(
 	}
 
 
-	protected function getFileIdForAlbums($fileId, $albums) {
+	/**
+	 * @param (\OCA\Photos\Album\AlbumInfo|mixed)[] $albums
+	 *
+	 * @psalm-param array<\OCA\Photos\Album\AlbumInfo|mixed> $albums
```

Comment:
What is the difference ?

Reply:
This is an automated psalm fix, it knows better than me :see_no_evil: